MycroftAI/mycroft-core
# Implement GitHub Actions for linting and unit tests

## Description
Build upon the work done in [PR 2764](https://github.com/MycroftAI/mycroft-core/pull/2764) to replace Travis CI with GitHub Actions.  Moved code coverage from coveralls to codecov due to some issues coveralls has with pytest and GitHub Actions.

## How to test
You can see the results of the GitHub Actions workflow in [draft PR 2817](https://github.com/MycroftAI/mycroft-core/pull/2817).  The "build" and "coverage" badges in README.md were replaced with GitHub Actions and codecov badges.

## Contributor license agreement signed?
CLA [Y]
